tests/aws/services/events/test_events.py

```diff
@@ -706,18 +706,113 @@ def _get_sqs_messages():
         assert message_body["time"] == "2022-01-01T00:00:00Z"
 
 
-class TestEventsEventBus:
+class TestEventBus:
     @markers.aws.validated
-    def test_create_custom_event_bus(self, aws_client, cleanups, snapshot):
+    @pytest.mark.skipif(
+        not is_v2_provider() and not is_aws_cloud(),
+        reason="V1 provider does not support this feature",
+    )
+    @pytest.mark.parametrize("regions", [["us-east-1"], ["us-east-1", "us-west-1", "eu-central-1"]])
+    def test_create_list_describe_delete_custom_event_buses(
+        self, aws_client_factory, regions, snapshot
+    ):
+        bus_name = f"test-bus-{short_uid()}"
+        snapshot.add_transformer(snapshot.transform.regex(bus_name, "<bus-name>"))
+
+        for region in regions:
+            events = aws_client_factory(region_name=region).events
```

Comment:
I guess this overwrites the randomized region and should work, but it's probably better to double-check this assumption. Just be aware that any other fixture-based client could use a randomized region. That's probably worth mentioning in a comment.

Reply:
I manually succesfully run the CircleCi pipeline with the flag `randomize-aws-credentials` set to True

Reply:
fixed template replacement of region in snapshots for setting a region different than the default region in: https://github.com/localstack/localstack/pull/10613/commits/82420ae50247da44042fa555fe3005cb5d0bacb4